### PR TITLE
Keep track of currently running checks

### PR DESCRIPTION
**Pull request by hush-hush**
### What does this PR do?
This will be very useful to troubleshoot slow or blocking checks

example:
```
curl -s http://localhost:5000/debug/vars | jq .runner.running
{
  "agent_telemetry:datadog.agent:35005bff9d65b1bd": "2021-04-16T14:26:37+02:00",
  "ntp:d884b5186b651429": "2021-04-16T14:26:37+02:00"
}

```

### CODE REVIEW

File: pkg/collector/runner/runner.go [hunk 1]
```diff
@@ -35,13 +35,20 @@ const (
...
-	TestWg      sync.WaitGroup
-	runnerStats *expvar.Map
-	checkStats  *runnerCheckStats
+	TestWg             sync.WaitGroup
+	runnerStats        *expvar.Map
+	runningChecksStats *expvar.Map
+	checkStats         *runnerCheckStats
 )
 
+type TimeVar struct{ t time.Time }
```
Comment (remeh) [inline pkg/collector/runner/runner.go:44]: I don't think using a struct is needed.

```go
type TimeVar time.Time

func (v *TimeVar) String() string { return fmt.Sprintf("\"%s\"", time.Time(v).Format(time.RFC3339)) }
```

Comment (remeh) [inline pkg/collector/runner/runner.go:44]:
```suggestion
type TimeVar time.Time
```

File: pkg/collector/runner/runner.go [hunk 2]
```diff
@@ -35,13 +35,20 @@ const (
...
+
+func (o *TimeVar) String() string { return fmt.Sprintf("\"%s\"", o.t.Format(time.RFC3339)) }
```
Comment (remeh) [inline pkg/collector/runner/runner.go:46]:
```suggestion
func (tv TimeVar) String() string { return fmt.Sprintf("\"%s\"", time.Time(tv).Format(time.RFC3339)) }
```

File: pkg/collector/runner/runner.go [hunk 3]
```diff
@@ -267,7 +274,9 @@ func (r *Runner) work() {
 		var err error
 		t0 := time.Now()
 
+		runningChecksStats.Set(string(check.ID()), &TimeVar{t: t0})
```
Comment (remeh) [inline pkg/collector/runner/runner.go:277]:
```suggestion
		runningChecksStats.Set(string(check.ID()), TimeVar(t0))
```

**Review by remeh — APPROVED** [review]

**Comment by sgnn7** [comment]
The ARM job is stuck (unrelated issue to PR) and the other tests pass so this PR is getting merged to ensure it's in before the freeze.

**Pull request merged**